### DISCUSSION TITLE
[Feature] Follow browser dark/light theme on Web

### DISCUSSION THREAD

**Discussion by Filip-Sutkowy** [immich-app/immich#3862] [Feature Request]
### The feature

Hi, immich has a light and dark theme switcher, it would be nice if there was an auto mode. I use app, that switch my system theme to light during the day and dark at night. For example, GitHub and YouTube have such a function. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
This has been implemented
